iree/test/e2e/regression/dynamic_dot_general.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: iree-run-mlir -export-all %s -iree-hal-target-backends=vmla -function-input="2x2xf32=[[1.0, 0.0], [0.0, 1.0]]" -function-input="2x3xf32=[[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]]" -function-input="2x2x2xf32=[[[1.0, 0.0], [0.0, 1.0]], [[2.0, 0.0], [0.0, 2.0]]]" -function-input="2x2x3xf32=[[[1.5, 2.5, 3.5], [4.5, 5.5, 6.5]], [[1.0, 2.0, 3.0], [4.0, 5.0, 6.0]]]" | IreeFileCheck %s
+// RUN: iree-run-mlir -export-all %s -iree-hal-target-backends=vmla -function-input="2x2xf32=[[1.0,0.0],[0.0,1.0]]" -function-input="2x3xf32=[[1.0,2.0,3.0],[4.0,5.0,6.0]]" -function-input="2x2x2xf32=[[[1.0,0.0],[0.0,1.0]],[[2.0,0.0],[0.0,2.0]]]" -function-input="2x2x3xf32=[[[1.5,2.5,3.5],[4.5,5.5,6.5]],[[1.0,2.0,3.0],[4.0,5.0,6.0]]]" | IreeFileCheck %s
```

Comment:
Are spaces load-bearing now? That worries me a bit.

Reply:
They always have been - tests that have spaces in them don't run on Windows. I have an issue for killing all our silly wrappers (see my screenshot of the ctest>bash>cmd>powershell>cmd>bash>bash>actual_test_binary.exe sandwich we create). I'd really like to be able to run tests, but also these tests should all be check tests because that's so much nicer :P

Reply:
Ah, a windows thing, not our parsing code.